src/common/translation/fi.json

```diff
@@ -465,5 +465,10 @@
   },
   "generic": {
     "all": "Kaikki"
+  },
+  "ticketValidation": {
+    "valid": "Lippu on voimassa",
+    "invalid": "Lippu ei ole voimassa",
+    "error": "Lipun tarkistaminen epäonnistui"
```

Comment:
The error comes when the enrolment is deleted or the enrolment reference id is spoofed. However, this might be the best text for it.